master/buildbot/newsfragments/waterfall-view-fixes.bugfix

```diff
@@ -0,0 +1 @@
+Minor fixes for waterfall view display issues and unit tests
```

Comment:
Could the fixes be described in more detail? The readers of the release notes will be interested in what changed, "minor fixes" doesn't tell them that. You can omit mentioning unit tests, this is an internal change.